railsconf/2020/let_juniors_fail.md

```diff
@@ -0,0 +1,36 @@
+# Give you juniors space to fail/Benefits of allowing junior developers to fail
+
+## Abstract
+
+In this talk, we will discuss the many different ways you can create an environment that promotes creativity and experimentation for junior developers. To create an environment that fosters mentorship, rapid growth, and respect, you must be willing to let your junior developers fail. Inversely, if you are a junior developer, you should seek out the type of environments that will not only allow, but encourage you to push push boundaries and learn from the resulting success or failure. This talk will focus on how to be a good mentor, and how to walk with your juniors instead of in front of them by learning to harness the unchecked optimism of your junior developers.
+
+## Details
+
+This talk is mainly aimed at mentors and junior developers. The main takeaway for mentors and senior developers is a better understanding, or gentle reminder, that they can help level up junior developers simply by getting out of their way and giving them the space to explore their ideas. For juniors, this talk will give them an insight into what a great mentor is, and what they should be looking for in a mentor.
+
+Outline:
+
+- Introduction
+  - My journey from junior to where I am now after only 2 years of experience of Rails
+  - Brief story about when my mentor, Nate Hopkins, gave me space to experiment and what an impact that had on my imposter syndrome and confidence
+- Explain what kind of space juniors need to grow
+  - Why failure is important
+  - Fostering mutual respect and creating a safe space
+  - Encouraging experimentation
+- Benefits of creating an environment that encourages creativity
+  - Mental health and burnout
+  - Turning failures into teaching moments
+  - Walking beside, instead of in front of, juniors
+- Conclusion
+
+## Pitch
+
+The more experience you have, the further removed you are from the needs of your junior developers. With only two years experience, I have a fresh perspective on the positive and negative mentorship tactics that many juniors are currently being exposed to. The main premise behind this talk comes from a discussion I had with my team lead about a pattern I had introduced that others didn’t seem to like. The takeaway from that conversation is my lead wanted to give me the space to explore my ideas. If I succeeded, we would have a great new pattern to use moving forward. If I failed, it wasn’t something we couldn’t recover from, and I would have learned a great deal. The key to these two possible outcomes is that regardless of whether I succeeded or failed in my attempt, my mentor was excited to help me realize the lessons I learned from the experiment that would help me to become a better developer moving forward.
```

Comment:
Awesome paragraph. The only thing missing IMHO is to hint at what Nate learned from working with you. For example, when you mentioned your personal chronology with jQuery, that was really helpful for me to apply the right lens.